wcfsetup/install/files/lib/system/form/builder/field/PasswordFormField.class.php

```diff
@@ -63,6 +63,12 @@ class PasswordFormField extends AbstractFormField implements
      */
     protected array $relatedFieldsId = [];
 
+    /**
+     * @var string[]
```

Comment:
This should be a `list<string>` instead because that is what the JS implementation expects. This affects the methods below too.